src/main/java/com/minecolonies/commands/ListColonies.java

```diff
@@ -0,0 +1,131 @@
+import java.util.List;
+
+/**
+ * List all colonies.
+ */
+public class ListColonies extends AbstractSingleCommand
+{
+
+    private static final String ID_TEXT                = "§2ID: §f";
+    private static final String NAME_TEXT              = "§2 Name: §f";
+    private static final String COORDINATES_TEXT       = "§2Coordinates: §f";
+    private static final String LIST_COMMAND_SUGGESTED = "/mc colonies list ";
+    private static final int    COLONIES_ON_PAGE       = 9;
+
+    /**
+     * Initialize this SubCommand with it's parents.
+     *
+     * @param parents an array of all the parents.
+     */
+    public ListColonies(@NotNull final String... parents)
+    {
+        super(parents);
+    }
+
+    @NotNull
+    @Override
+    public String getCommandUsage(@NotNull final ICommandSender sender)
+    {
+        return super.getCommandUsage(sender) + "";
+    }
+
+    @Override
+    public void execute(@NotNull final MinecraftServer server, @NotNull final ICommandSender sender, @NotNull final String... args) throws CommandException
+    {
+        int page = 1;
+        final List<Colony> colonies = ColonyManager.getColonies();
+        final int colonyCount = colonies.size();
+
+        // check to see if we have to add one page to show the half page
+        final int halfPage = (colonyCount % COLONIES_ON_PAGE == 0) ? 0 : 1;
+        final int pageCount = ((colonyCount) / COLONIES_ON_PAGE) + halfPage;
+
+        if (args.length != 0)
+        {
+            try
+            {
+                page = Integer.parseInt(args[0]);
+            }
+            catch (NumberFormatException e)
+            {
+                //ignore and keep page 1.
+            }
+        }
+        if (page < 1 || page > pageCount)
+        {
+            page = 1;
+        }
+
+        final int pageStartIndex = COLONIES_ON_PAGE * (page - 1);
+        final int pageStopIndex = Math.min(COLONIES_ON_PAGE * page, colonyCount);
+        final int prevPage = Math.max(0, page - 1);
+        final int nextPage = Math.min(page + 1, (colonyCount / COLONIES_ON_PAGE) + halfPage);
+
+        List<Colony> coloniesPage;
+
+        if (pageStartIndex < 0 || pageStartIndex >= colonyCount)
+        {
+            coloniesPage = new ArrayList<>();
+        }
+        else
+        {
+            coloniesPage = colonies.subList(pageStartIndex, pageStopIndex);
+        }
+
+        final ITextComponent headerLine = new TextComponentString("§2   ------------------ page " + page + " of " + pageCount + " ------------------");
+        sender.addChatMessage(headerLine);
+
+        for (final Colony colony : coloniesPage)
+        {
+            sender.addChatMessage(new TextComponentString(ID_TEXT + colony.getID() + NAME_TEXT + colony.getName()));
+            final BlockPos center = colony.getCenter();
+            sender.addChatMessage(new TextComponentString(COORDINATES_TEXT + String.format("§4x=§f%s §4y=§f%s §4z=§f%s", center.getX(), center.getY(), center.getZ())));
```

Comment:
Example: add the other part of this to the COORDINATES_TEXT and do String.format(COORDINATES_TEXT, ....)